### ISSUE TITLE
autoflip 0x00d36c4

### ISSUE THREAD

**Issue by kim-minsol** [google/mediapipe#3398]
<em>Please make sure that this is a [solution](https://google.github.io/mediapipe/solutions/solutions.html) issue.<em>

**System information** (Please provide as much relevant information as possible)
- Have I written custom code (as opposed to using a stock example script provided in Mediapipe):

bazel-out\x64_windows-opt\bin\mediapipe\examples\desktop\autoflip\run_autoflip --calculator_graph_config_file=mediapipe/examples/desktop/autoflip/autoflip_graph.pbtxt --input_side_packets=input_video_path=mediapipe/examples/desktop/autoflip/life_at_google.mp4,output_video_path=mediapipe/examples/desktop/autoflip/life_at_google_autofliped_v1.mp4,aspect_ratio=1:1

- [MediaPipe version](https://github.com/google/mediapipe/releases):
- Bazel version: 5.1.1
- Solution (e.g. FaceMesh, Pose, Holistic): autoflip

The first time I ran AutoFlip, I got a well-cropped, well-played video. However, the error continues to occur after that. When I got the right video, ffmpeg and open264 lib was not installed. And the error appears even after installing ffmpeg, open264 lib.

The mediapipe git clone is also new(I did it today), and the error continues to occur even when the bazel is newly installed.

Please help me.... I really need your help.

**Standalone code you may have used to try to get what you need :**
![image](https://user-images.githubusercontent.com/81224613/172003636-30c8c9c6-6420-4e24-b795-2556ec790b60.png)

**Other info / Complete Logs :**
Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached:

set BAZEL_VS=C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools
set BAZEL_VC=C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC
set BAZEL_VC_FULL_VERSION=14.29.30133
set BAZEL_WINSDK_FULL_VERSION=10.0.19041.0

bazel build -c opt --define MEDIAPIPE_DISABLE_GPU=1 --action_env PYTHON_BIN_PATH="C://Users//kkhhj//AppData//Local//Programs//Python//Python38-32//python.exe" mediapipe/examples/desktop/autoflip:run_autoflip

I changed ceres.bazel because of the issue #2985
C:\Users\kkhhj\AppData\Roaming\SPB_Data\_bazel_kkhhj\jbcms7g5\external\ceres_solver\bazel\ceres.bzl

cmd:
C:\Users\kkhhj\mediapipe>bazel-out\x64_windows-opt\bin\mediapipe\examples\desktop\autoflip\run_autoflip --calculator_graph_config_file=mediapipe/examples/desktop/autoflip/autoflip_graph.pbtxt --input_side_packets=input_video_path=mediapipe/examples/desktop/autoflip/life_at_google.mp4,output_video_path=mediapipe/examples/desktop/autoflip/life_at_google_autofliped_v1.mp4,aspect_ratio=1:1
'ffmpeg'은(는) 내부 또는 외부 명령, 실행할 수 있는 프로그램, 또는
배치 파일이 아닙니다.
ls: cannot access 'C:\Users\kkhhj\AppData\Local\Temp\sals.0': No such file or directory

Failed to load OpenH264 library: openh264-1.7.0-win64.dll
        Please check environment and/or download library: https://github.com/cisco/openh264/releases

[libopenh264 @ 00000204954bd200] Incorrect library version loaded
Could not open codec 'libopenh264': Unspecified error

C:\Users\kkhhj\mediapipe>

**Comment by sureshdagooglecom** [comment]
Hi @kim-minsol ,
Could you provide steps to reproduce this issue.

**Comment by kim-minsol** [comment]
> Hi @kim-minsol , Could you provide steps to reproduce this issue.

bazel build -c opt --define MEDIAPIPE_DISABLE_GPU=1 --action_env PYTHON_BIN_PATH="C://Users//kkhhj//AppData//Local//Programs//Python//Python38-32//python.exe" mediapipe/examples/desktop/autoflip:run_autoflip

1. I built autoflip with this code, but the issue #2985 occurred during the process.

2. To resolve the error, Wno-sign-compare was removed from the ceres.bzl part.

3. After the build, autoflip ran well and got the video properly. (life_at_google.mp4)
However, the log that ffmpeg is not running and the version of openh264 lib is not correct was app was shown, so I installed it.

4. After installation, I tried to run the same video again with input, but 0x00d36c4 error occurred.

5. After removing ffmpeg and openh264, I tried to run it with the same and different videos, but I get the corresponding error.

6. (+) I rebuilt bazel and mediapipe git clone and succeeded in getting a proper video.

7. (+) However, if you run the same image twice with input, the corresponding error (0x00d36c4) occurs again.

I'm curious about the reason why the video is broken and the solution. I need help!
